build_service/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright 2020 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+export http_proxy_host=$(echo $http_proxy | awk -F':' {'print $2'} | tr -d '/')
+export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
+export https_proxy_port=$(echo $https_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$https_proxy_port"
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+
+REMOTE_URL=https://github.com/secure-device-onboard/epid-verification-service.git
+REMOTE_BRANCH=master
+
+if [ "$use_remote" = "1" ]; then
+    echo "Building $REMOTE_URL : $REMOTE_BRANCH"
+    cd /tmp/
+    git clone $REMOTE_URL
+    cd /tmp/epid-verification-service/
```

Comment:
Remote repo may not be named epid-verification-service (especially if it's a fork).
I recommend the 'git clone <remote> <local>' form so you have control over the name of the local directory.

Reply:
updated

Reply:
Where do we create the folder /tmp/epid-verification-service? Without that, this might fail.

Better way to do this would be:
cd /tmp
git clone -b $REMOTE_BRANCH $REMOTE_URL epid-verification-service